### PR TITLE
Loop

### PR DESCRIPTION
**Pull request by lurkio**
Pics invisible till loaded and colours set up. 
Loop back to menu on keypress.

### CODE REVIEW

File: basic/ani.txt [hunk 1]
```diff
@@ -22,10 +23,8 @@ M%=M%+1
 NEXT
 D%=?M%
 M%=M%+1
-K%=INKEY(D%-2)
-NEXT
-M%=N%
-UNTIL 0
+K%=INKEY(D%-2):IFK%=-1NEXT:M%=N%:UNTIL0
+CLS:RUN
```
Comment (lurkio) [inline basic/ani.txt]: Clear screen so you don't see residual old image data when you next switch back to MODE2

Comment (lurkio) [inline basic/ani.txt:27]: Clear screen so you don't see residual old image data when prog next switches back to MODE2.

File: basic/ani.txt [hunk 2]
```diff
@@ -1,9 +1,10 @@
-MODE7
+MODE7:PRINT:*FX21,0
 HIMEM=HIMEM-&200
-PROCmenu
+PROCmenu:CLS
```
Comment (lurkio) [inline basic/ani.txt:3]: Clear screen so you don't see residual old image data when prog next switches back to MODE7.

File: basic/ani.txt [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
 MODE2
 HIMEM=HIMEM-&100
 VDU 23,1,0;0;0;0;
+FORC%=0TO15:VDU19,C%,0;0;:NEXT
```
Comment (lurkio) [inline basic/ani.txt:7]: Niceness.

**Comment by mungre** [comment]
Thanks, this is much nicer.

**Pull request merged**